testing/tests/DevExpress.ui.widgets/listParts/commonTests.js

```diff
@@ -4030,3 +4031,81 @@ if(devices.real().deviceType === 'desktop') {
         });
     });
 }
+
+if(QUnit.urlParams['nojquery']) {
+    QUnit.module('ShadowDOM', {
+    }, () => {
+        QUnit.test('ShadowDom: drag item', function(assert) {
+            const root = document.querySelector('#list');
+            const container = document.createElement('div');
+            const list = document.createElement('div');
+
+            root.attachShadow({ mode: 'open' });
+            container.appendChild(list);
+            root.shadowRoot.appendChild(container);
+
+            const $list = $(list).dxList({
+                items: ['One', 'Two', 'Three'],
+                itemDragging: { allowReordering: true },
+                focusStateEnabled: true,
+            });
+
+            const $items = $list.find(toSelector(LIST_ITEM_CLASS));
+
+            const pointer = reorderingPointerMock($items.first());
+
+            pointer.dragStart().drag(34);
+            pointer.dragEnd();
+
+            const orderedItems = $list.find(toSelector(LIST_ITEM_CLASS)).toArray().map(e => e.innerText.trim());
+
+            assert.deepEqual(orderedItems, ['Two', 'Three', 'One']);
+
+            $(container).empty();
```

Comment:
it seems to be excess

Reply:
We need this row here, since we need to remove all elements to prevent the `clearTimeouts` error. In does not work by default, since jQuery cannot walk through ShadowRoot when it's clearing timeouts

Reply:
oh, okay. Then we need move it to `afterEach` and add a comment like:
`TODO: get rid of it after fix jquery event bubbling to shadow dom`